include/erlcloud_ddb2.hrl

```diff
@@ -309,6 +309,24 @@
          time_to_live_description :: undefined | #ddb2_time_to_live_description{}
         }).
 
+-record(ddb2_transact_write_items,
+        {consumed_capacity :: undefined | [#ddb2_consumed_capacity{}],
+         item_collection_metrics :: undefined | [{erlcloud_ddb2:table_name(), [#ddb2_item_collection_metrics{}]}],
+         % AWS documentation infer that it should be possible to get old return
+         % values upon condition failure, but I have been unable to do so.
+         % Still, let's put a field so we can return {ok, []} on success.
+         attributes :: undefined
+        }).
+
+-record(ddb2_transact_get_items_response,
```

Comment:
i wonder is the name is miss-leading...as this is a single item

Reply:
Maybe it would just be called [`#ddb2_item_response`](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ItemResponse.html) per the [Responses doc](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactGetItems.html#API_TransactGetItems_ResponseSyntax)